etna/core/middleware.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import settings
+from django.template.response import SimpleTemplateResponse
+
+from etna.ciim.utils import get_date_for_retry_after_header
+
+
+def get_client_ip(request) -> str:
+    x_forwarded_for = request.META.get("HTTP_X_FORWARDED_FOR")
+    if x_forwarded_for:
+        ip = x_forwarded_for.split(",")[0]
+    else:
+        ip = request.META.get("REMOTE_ADDR")
+    return ip
```

Comment:
Any reason why this method is added here, but `get_date_for_retry_after_header()` is added to utils?

Reply:
get_date_for_retry_after_header() is removed and refactored in middleware.py